src/models/github-server-app.ts

```diff
@@ -65,7 +65,7 @@ export class GitHubServerApp extends EncryptedModel {
 
 		return this.findOne({
 			where: {
-				id: gitHubServerAppId
+				appId: gitHubServerAppId
```

Comment:
@mboudreau 
I got a feeling the previous usage of `id` column is incorrect. We should be using `appId`, as it is the one that github recognised. 

Has to change this to make the `github-route.test.ts` works anyway.

Reply:
No it's meant to be id. When we call `await GitHubServerApp.getForGitHubServerAppId(gitHubAppId);` we are after the unique `id` not the `appId` as appIds may be repeated across servers.

Reply:
I know it's the one that github recognizes, but it's not unique in the GHE table as you can have collisions between appIds for each server. We need to use the PK for this.

Reply:
it's just to get the object, then when you do a call out to the github api, you get the appId, but you're also calling a specific server

Reply:
No, it's meant to be `id`. When we call `await GitHubServerApp.getForGitHubServerAppId(gitHubAppId);` we are passing in the unique `id` that is created when an entry in added to the table. `appId` is not unique as it may be repeated across servers. Happy for you to rename that param if you think it will make it clear but it's meant to be referencing the id of the target entry in the GitHub Server App table (hence gitHubServerAppId).

Reply:
Okay, thanks guys, yeah, I was confused about the `appId` in `GitHubServerApp` table is not the `gitHubAppId` everywhere. I'll revert the change of this bit, think about the naming in another PR.

Reply:
Reverted and fixed. Closing this thread.